drivers/wifi/nrf700x/zephyr/inc/zephyr_coex.h

```diff
@@ -23,49 +23,54 @@ enum nrf_wifi_pta_wlan_op_band {
 
 /**
  * @function   nrf_wifi_coex_config_pta(enum nrf_wifi_pta_wlan_op_band wlan_band,
- *             bool separate_antennas)
+ *             bool antenna_mode, bool ble_role, bool wlan_role)
  *
  * @brief      Function used to configure PTA tables of coexistence hardware.
  *
  * @param[in]  enum nrf_wifi_pta_wlan_op_band wlan_band
- * @param[in]  separate_antennas
+ * @param[in]  antenna_mode
```

Comment:
Using a boolean type and calling it `antenna_mode` doesn't make sense, `mode` implies a `enum`, IMHO, original naming is better for a bool unless you want to make it an `enum`.